client/packages/requisitions/src/RnRForms/DetailView/ContentArea.tsx

```diff
@@ -56,17 +57,16 @@ export const ContentArea = ({
   const t = useTranslation();
   const containerRef = useRef<HTMLDivElement>(null);
   const listRef = useRef<ViewportListRef>(null);
+  const { setListRef } = useRnRFormContext(
```

Comment:
As per PR description, mainly relocated search logic because need to scroll to the error record on finalise, so that all got moved and i decided to also move search functionality to the context

Reply:
Why oneTime ? I noticed that ContentArea was refreshed when some other state in RnRFormContext was changed, didn't investigate further but oneTime works (also added to DetailView, just in case)